hs_app_netCDF/receivers.py

```diff
@@ -20,58 +21,61 @@
 from hs_file_types.models.netcdf import create_header_info_txt_file, add_metadata_to_list
 
 
-# receiver used to extract metadata after user click on "create resource"
-@receiver(pre_create_resource, sender=NetcdfResource)
-def netcdf_pre_create_resource(sender, **kwargs):
-    files = kwargs['files']
-    metadata = kwargs['metadata']
+@receiver(post_create_resource, sender=NetcdfResource)
+def netcdf_post_create_resource(sender, **kwargs):
```

Comment:
Since now you have a post resource creation handler, some of the unit tests for netcdf resource type needs send the post create signal.